pitkeel-py/daemon.py

```
@@ -111,12 +111,32 @@ def _shutdown(dry_run: bool = False) -> None:
 
-    # Literal OS shutdown — visceral by design [E1 spec, M1]
-    try:
-        subprocess.run(["shutdown", "now"], timeout=10)
-    except (FileNotFoundError, subprocess.TimeoutExpired):
-        sys.stderr.write("PITKEEL: shutdown command failed\n")
-        sys.exit(1)
+    # Literal OS shutdown - visceral by design [E1 spec, M1]
+    # Check exit code and retry once on failure before giving up.
+    for attempt in range(2):
+        try:
+            result = subprocess.run(["shutdown", "now"], timeout=10)
```

Ruff (0.15.5)

[error] Starting a process with a partial executable path

(S607)